renovate.json

```diff
@@ -11,5 +11,19 @@
     "gomodTidy",
     "gomodUpdateImportPaths"
   ],
-  "suppressNotifications": ["prEditedNotification"]
+  "suppressNotifications": ["prEditedNotification"],
+  "packageRules": [
+    {
+      "matchFileNames": [
+        "docker-compose/cassandra/v3/docker-compose.yaml",
+        "docker-compose/cassandra/v4/docker-compose.yaml",
+        "docker-compose/elasticsearch/v7/docker-compose.yml",
+        "docker-compose/elasticsearch/v8/docker-compose.yml",
+        "docker-compose/opensearch/v1/docker-compose.yml",
+        "docker-compose/opensearch/v2/docker-compose.yml"
+      ],
+      "matchUpdateTypes": ["major"],
```

Comment:
is it possible to also suppress patches? The bot just opened #5620 with `7.17.21 -> 7.17.22`, which is complete noise for e2e test dependencies.

Reply:
For just the same files? We can add "patch" to the "matchUpdateTypes" if it is just for those 6 files, or create a new packageRules for the files to exclude.
https://docs.renovatebot.com/configuration-options/#matchupdatetypes

Reply:
yes, same files

Reply:
I created #5622 to fix this